plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/ServicesAndRoutingPage.java

```diff
@@ -92,11 +103,40 @@ protected void doCreateControls(Composite parent, DataBindingContext dbc) {
 		btnAddRoute.setText("Add Route");
 		btnAddRoute.setToolTipText("Adding a route to the service will make the image accessible\noutside of the OpenShift cluster on all the available service ports. \nYou can target a specific port by editing the route later.");
 		GridDataFactory.fillDefaults()
-			.align(SWT.FILL, SWT.FILL).grab(false, false).applyTo(btnAddRoute);
+			.align(SWT.FILL, SWT.FILL).grab(false, false).span(2, 1).applyTo(btnAddRoute);
 		ValueBindingBuilder.bind(WidgetProperties.selection().observe(btnAddRoute))
 			.to(BeanProperties.value(IServiceAndRoutingPageModel.PROPERTY_ADD_ROUTE)
 			.observe(model))
 			.in(dbc);
+		
+		Label labelRouteHostname = new Label(routingContainer, SWT.NONE);
+		labelRouteHostname.setText("Hostname:");
+	    GridDataFactory.fillDefaults()
+          .align(SWT.FILL, SWT.FILL)
+          .applyTo(labelRouteHostname);
+
+		Text textRouteHostname = new Text(routingContainer, SWT.BORDER);
+        GridDataFactory.fillDefaults()
+        .align(SWT.FILL, SWT.CENTER)
```

Comment:
"Hostname" label is not centered vertically when compared to the text field.
![image](https://cloud.githubusercontent.com/assets/25126/19072444/7ff70a36-8a35-11e6-8c8b-3d06f23f186f.png)